code/modules/antagonists/clockcult/clock_helpers/power_helpers.dm

```diff
@@ -6,19 +6,20 @@
 /proc/adjust_clockwork_power(amount) //Adjusts the global clockwork power by this amount (min 0.)
 	if(GLOB.ratvar_approaches)
 		amount *= 0.75 //The herald's beacon reduces power costs by 25% across the board!
-	GLOB.clockwork_power = GLOB.ratvar_awakens ? INFINITY : max(0, GLOB.clockwork_power + amount)
-	GLOB.clockwork_power = CLAMP(GLOB.clockwork_power, 0, MAX_CLOCKWORK_POWER)
+	if(GLOB.ratvar_awakens)
+		GLOB.clockwork_power = INFINITY
```

Comment:
if there is an issue with glob.clockwork power being more than max clockwork power other than display wackiness better tell me someone @Xhuis